compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/layer/SkiaGraphicsLayer.skiko.kt

```diff
@@ -536,6 +489,14 @@ actual class GraphicsLayer internal constructor(
         ImageBitmap(size.width, size.height).apply { draw(Canvas(this), null) }
 }
 
+
+private fun SkCanvas.clipOutline(outline: Outline?, bounds: SkRect) = when (outline) {
+    is Outline.Rectangle -> clipRect(outline.rect.toSkiaRect(), antiAlias = true)
+    is Outline.Rounded -> clipRRect(outline.roundRect.toSkiaRRect(), antiAlias = true)
+    is Outline.Generic -> clipPath(outline.path.asSkiaPath())
```

Comment:
```suggestion
    is Outline.Generic -> clipPath(outline.path.asSkiaPath(), antiAlias = true)
```
In the previous code it was with antialias:
```
    override fun clipPath(path: Path, clipOp: ClipOp) {
        val antiAlias = true
        skia.clipPath(path.asSkiaPath(), clipOp.toSkia(), antiAlias)
    }
```